### ISSUE TITLE
Publish migration

### ISSUE THREAD

**Issue by philipmclifton** [TheDragonCode/laravel-migration-actions#47]
### Description:
Is it possible to publish the migration file?

**Comment by andrey-helldar** [comment]
In general, yes, the usual [`copy`](https://www.php.net/manual/en/function.copy) operation. But why?

**Comment by philipmclifton** [comment]
The migration to create the table does not appeare in the migrations table.

When you do: migrate:refresh the table is not dropped and recreated.

It’s a bit annoying when your are refreshing often to have to manually clear the migrations_actions table

Phil

**Comment by andrey-helldar** [comment]
I understood what the problem is.

At the moment, the mechanism of this package uses the mechanism of migrations with some changes.

The peculiarity of migrations is that it does not create a table to create a table for storing migration statuses.

In the near future I will add the ability to automatically create a migration table when executing the `art migrate` command and its helpers.

**Comment by andrey-helldar** [comment]
Have you tried [`php artisan migrate:actions:refresh`](https://github.com/TheDragonCode/laravel-migration-actions#roll-back--action-using-a-single-command) command?

**Comment by andrey-helldar** [comment]
@philipmclifton, I added a `migrate:actions:fresh` command to quickly restart actions on a clean base.

In case of rolling back migrations, the startup script can be one of the options:

1.
```bash
php artisan migrate:fresh
php artisan migrate:actions:fresh
```

2.
```bash
php artisan migrate:refresh
php artisan migrate:actions:fresh
```

The [`migrate:actions:fresh`](https://github.com/TheDragonCode/laravel-migration-actions#drop-all-actions--migrate-actions) command released in version [`2.8.0`](https://github.com/TheDragonCode/laravel-migration-actions/releases/tag/v2.8.0).
